src/server/game/Scripting/ScriptMgr.h

```diff
@@ -841,6 +841,9 @@ class PlayerScript : public ScriptObject
     // Called when a player is added to battleground
     virtual void OnAddToBattleground(Player* /*player*/, Battleground* /*bg*/) { }
 
+    // Called when a player queues a Random Dungeon using the RDF (Random Dungeon Finder)
```

Comment:
Need new style info

Reply:
https://www.azerothcore.org/wiki/how-to-document-code

Reply:
sorry I missed this, I'll address this in another PR